components/canvas/raqote_backend.rs

```diff
@@ -183,8 +184,9 @@ impl Path {
         unimplemented!()
     }
 
-    pub fn contains_point(&self, _x: f64, _y: f64, _path_transform: &Transform2D<f32>) -> bool {
-        unimplemented!()
+    pub fn contains_point(&self, x: f64, y: f64, _path_transform: &Transform2D<f32>) -> bool {
+        let path = self.as_raqote();
+        path.contains_point(0.1, path.winding, x as f32, y as f32)
```

Comment:
The newest version of raqote drops the winding parameter from contains_point() (though you can just deal with that later)